e2e/forbidden_annotations_regex_test.go

```diff
@@ -0,0 +1,85 @@
+// +build e2e
+
+// Copyright 2020-2021 Clastix Labs
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	capsulev1alpha1 "github.com/clastix/capsule/api/v1alpha1"
```

Comment:
Why do we need the old `capsulev1alpha1` import here?

All tests must be addressed on the stored one, `v1beta1`.